tools/node-modules

```diff
@@ -132,6 +132,23 @@ EOF
     fi
 }
 
+# called from tools/test-static-code
+cmd_test_static_code() {
+    node_modules_update=$(git log --format=%H -n 1 node_modules)
```

Comment:
This commit "tools/test-static-code: add node_modules check" adds the same `cmd_test_static_code()` *again*, it is already present.